libmachine/drivers/plugin/localbinary/plugin.go

```diff
@@ -139,6 +139,10 @@ func (lbe *Executor) Start() (*bufio.Scanner, *bufio.Scanner, error) {
 
 	// #nosec G204
 	lbe.cmd = exec.Command(lbe.binaryPath)
+	lbe.cmd.Env = append([]string{
```

Comment:
what is happening that this might be needed?

Reply:
After a crc run, PluginEnvKey seems to be still defined in the current environment and further crc commands fail (because crc thinks it is started as a plugin).

I don't understand why this PluginEnvKey is still defined after crc exited. It's weird..

Reply:
Can we add some more env debugging to a fake PR to understand more what's going on before going forward with this change? Or is this hard to trigger in CI?